docs/docs/flink.md

```diff
@@ -401,4 +401,3 @@ There are some features that are do not yet supported in the current Flink Icebe
 * Don't support creating iceberg table with hidden partitioning. [Discussion](http://mail-archives.apache.org/mod_mbox/flink-dev/202008.mbox/%3cCABi+2jQCo3MsOa4+ywaxV5J-Z8TGKNZDX-pQLYB-dG+dVUMiMw@mail.gmail.com%3e) in flink mail list.
 * Don't support creating iceberg table with computed column.
 * Don't support creating iceberg table with watermark.
```

Comment:
Side note: This is also not true anymore: https://github.com/apache/iceberg/pull/12191

Reply:
shall we remove that in this PR too? or open a new one?

Reply:
yes, makes sense. So the two remaining features to list as not implemented are:

* Don't support creating iceberg table with hidden partitioning. [Discussion](http://mail-archives.apache.org/mod_mbox/flink-dev/202008.mbox/%3cCABi+2jQCo3MsOa4+ywaxV5J-Z8TGKNZDX-pQLYB-dG+dVUMiMw@mail.gmail.com%3e) in flink mail list.
* Don't support creating iceberg table with computed column.

is that right?

Reply:
Yes @rmoff, those are the 2 last missing features mentioned here.
Could you please update the doc accordingly?

Thanks,
Peter

Reply:
Fixed d60a9a8c94d9bf42c27a4545dd9ed50c36e617e2

Reply:
@pvary could you review/merge please? thanks.